### PR TITLE
Restructured project folders

### PR DESCRIPTION
**Pull request by thisistrivial**

### CODE REVIEW

File: src/main/java/rocks/spacecalamity/app/ipc/Client.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package rocks.spacecalamity.app.ipc;
+
+import java.io.PrintWriter;
+
+abstract class Client {
```
Comment (thisistrivial) [inline src/main/java/rocks/spacecalamity/app/ipc/Client.java:5]: Client object parents types of clients (i.e. storage and users)

File: src/main/java/rocks/spacecalamity/app/ipc/ClientThread.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.PrintWriter;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+
+class ClientThread extends Thread {
```
Comment (thisistrivial) [inline src/main/java/rocks/spacecalamity/app/ipc/ClientThread.java:11]: ClientThread acts as a thread to handle a new client

File: src/main/java/rocks/spacecalamity/app/ipc/ConnectionHandler.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.PrintWriter;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+public class ConnectionHandler {
```
Comment (thisistrivial) [inline src/main/java/rocks/spacecalamity/app/ipc/ConnectionHandler.java:13]: ConnectionHandler handles all IPC stuff

File: src/main/java/rocks/spacecalamity/app/ipc/ClientThread.java [hunk 4]
```diff
@@ -52,6 +52,8 @@ public void run() {
       return;
     }
     try {
+      this.writer.println("connected");
+      this.writer.flush();
```
Comment (m00s3-3ggs) [inline src/main/java/rocks/spacecalamity/app/ipc/ClientThread.java:56]: ha theres a function called flush

**Review by m00s3-3ggs — APPROVED** [review]
Package restructuring successful, looks good!

**Pull request merged**